src/Common/src/System/CodeDom/CodeObject.cs

```diff
@@ -19,8 +19,6 @@ internal class CodeObject
     {
         private IDictionary _userData = null;
 
-        public CodeObject() { }
```

Comment:
This is not dead code.  It's public surface area.  This needs to be reverted.

Reply:
Reverted. Sorry for the noise!